frontend/services/src/tickets/ticketService.ts

```diff
@@ -0,0 +1,19 @@
+import { TicketId } from "@/interfaces/EventTypes";
+import { EMPTY_TICKET, Ticket, TicketCollectionPath } from "@/interfaces/TicketTypes";
+import { Logger } from "@/observability/logger";
+import { db } from "@/services/src/firebase";
+import { doc, getDoc } from "firebase/firestore";
+
+const ticketServiceLogger = new Logger("ticketServiceLogger");
+
+export async function getTicketById(ticketId: TicketId): Promise<Ticket> {
+  ticketServiceLogger.info(`getTicketById, ${ticketId}`);
+  try {
+    const ticket = await getDoc(doc(db, TicketCollectionPath, ticketId));
+    const ticketData = ticket.data() as Ticket;
+    return { ...EMPTY_TICKET, ...ticketData, ticketId: ticketId };
```

Comment:
what's the reason for needing the EMPTY_TICKET here? should `ticketData` retrieved from the database already have all the required fields

Reply:
same goes for EMPTY_ORDER as well